repos/spack_repo/builtin/packages/py_pylint/package.py

```diff
@@ -28,6 +28,8 @@ class PyPylint(PythonPackage):
 
     license("GPL-2.0-or-later")
 
+    version("3.3.7", sha256="2b11de8bde49f9c5059452e0c310c079c746a0a8eeaa789e5aa966ecc23e4559")
+    # VERSIONS in-between not checked - all need different versions of dependencies!
```

Comment:
Can you explain what this comment means?

Reply:
The latest version before was 2.16.1
And between 3.3.7 and 2.16.1, there were a lot of other releases.
And these all have different version constraints to their dependencies.
So I didn't want to track all these.